### PR TITLE
.NET Interactive: Enable_try_dotnet_to_batch_8c

### PR DESCRIPTION
**Pull request by v-mepa**
**Enable_try_dotnet_to_batch_8c**

URL | Comments
-- | --
https://docs.microsoft.com/en-us/dotnet/api/system.char.tostring | Duplicate samples for all overloads? Potentially   move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.char.iswhitespace | Duplicate samples for all overloads? Potentially   move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.char.iscontrol | **Only enable for first example. Second example   might be culture or OS dependent?**
https://docs.microsoft.com/en-us/dotnet/api/system.char.tolower | Duplicate samples for all overloads? Potentially   move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.char.isletterordigit | Duplicate samples for all overloads? Potentially   move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.char.issymbol | Duplicate samples for all overloads? Potentially   move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.char.ispunctuation | Duplicate samples for all overloads? Potentially   move it to the MemberGroup
https://docs.microsoft.com/en-us/dotnet/api/system.char.converttoutf32 | Duplicate samples for all overloads? Potentially   move it to the MemberGroup

### CODE REVIEW

**Comment by v-mepa** [comment]
fyi @mairaw, let me know how this looks :)

File: xml/System/Char.xml [hunk 1]
```diff
@@ -4945,9 +4879,9 @@ This member is an explicit interface member implementation. It can be used only
 ## Examples
  The following code example demonstrates <xref:System.Char.ToString%2A>.
 
- [!code-cpp[System.Char.ToString#17](~/samples/snippets/cpp/VS_Snippets_CLR_System/system.Char.ToString/CPP/tostring.cpp#17)]
- [!code-csharp[System.Char.ToString#17](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Char.ToString/CS/tostring.cs#17)]
- [!code-vb[System.Char.ToString#17](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Char.ToString/VB/tostring.vb#17)]
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR_System/system.Char.ToString/CPP/tostring.cpp" id="Snippet17":::
```
Comment (mairaw) [inline xml/System/Char.xml:4882]: This one was not added to the MemberGroup element

File: xml/System/Char.xml [hunk 2]
```diff
@@ -4737,14 +4682,12 @@ This member is an explicit interface member implementation. It can be used only
 
  Use <xref:System.String.ToLower%2A?displayProperty=nameWithType> to convert a string to lowercase.
 
-
-
 ## Examples
  The following example demonstrates <xref:System.Char.ToLower%2A>.
```
Comment (mairaw) [inline xml/System/Char.xml:4686]: This also needs to be moved to the MemberGroup element

File: xml/System/Char.xml [hunk 3]
```diff
@@ -490,14 +490,12 @@ When a managed <xref:System.Char> type, which is represented as a Unicode UTF-16
...
 
-
-
 ## Examples
  The following code example demonstrates the <xref:System.Char.ConvertToUtf32%2A> and <xref:System.Char.ConvertFromUtf32%2A> methods.
 
- [!code-cpp[char.cvtutf32#1](~/samples/snippets/cpp/VS_Snippets_CLR/char.cvtutf32/CPP/utf.cpp#1)]
- [!code-csharp[char.cvtutf32#1](~/samples/snippets/csharp/VS_Snippets_CLR/char.cvtutf32/CS/utf.cs#1)]
- [!code-vb[char.cvtutf32#1](~/samples/snippets/visualbasic/VS_Snippets_CLR/char.cvtutf32/VB/utf.vb#1)]
+ :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR/char.cvtutf32/CPP/utf.cpp" id="Snippet1":::
```
Comment (mairaw) [inline xml/System/Char.xml:496]: this needs to be moved to the MemberGroup element

Comment (v-mepa) [reply xml/System/Char.xml]: Done

File: xml/System/Char.xml (hunk 2 above)
Comment (v-mepa) [reply xml/System/Char.xml]: done

File: xml/System/Char.xml (hunk 1 above)
Comment (v-mepa) [reply xml/System/Char.xml]: done

**Pull request merged**